### DISCUSSION TITLE
Api

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#187] [Q&A] [answered]
https://github.com/codeman1o1/TIJK-Bot/blob/main/cogs/api.py

De Api was er toch uitgehaald?

**Comment by karstenzip** [comment]
Are you maybe confused with the AI chatbot?

**Comment by karstenzip** [comment]
@JustIanJ 
Please answer

**Comment by JustIanJ** [comment]
no

**Reply by JustIanJ** [reply]
i mean yes

**Comment by karstenzip** [comment, marked as answer]
Okay, well the AI chatbot was a beta feature, but it got removed later
